corehq/apps/reports/util.py

```diff
@@ -70,7 +70,7 @@ def get_group(group='', **kwargs):
     return group
 
 
-def get_all_users_by_domain(domain=None, group=None, user_ids=None,
+def get_all_users_by_domain(domain=None, group=None, user_ids=[],
```

Comment:
check out 9this link](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments) as to why mutable default arguments are generally not recommended